src/components/container/IntradayHeartRateChart/IntradayHeartRateChart.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useContext } from 'react'
+import { DateRangeContext } from '../../presentational/DateRangeCoordinator/DateRangeCoordinator'
+import { add, startOfDay } from 'date-fns'
+import { LayoutContext, LoadingIndicator } from '../../presentational'
+import { HalfDayData, FullDayData } from "./IntradayHeartRateChart.data"
+import { ColorDefinition, resolveColor } from '../../../helpers/colors'
+import { useInitializeView } from '../../../helpers/Initialization'
+import { DeviceDataV2Aggregate, DeviceDataV2AggregateQuery, DeviceDataV2Namespace } from '@careevolution/mydatahelps-js'
+import { hrIntradayDataProvider } from '../../../helpers/heart-rate-data-providers'
+import LineChart, { LineChartAxis, LineChartDataPoint } from '../../presentational/LineChart/LineChart'
+
+export type AggregationOption = "avg" | "count" | "min" | "max" | "sum";
+export type IntradayHeartRatePreviewState = "Default" | "CompleteDataWithThresholds" | "PartialDataWithThresholds" | "NoData";
+
+export interface IntradayChartThreshold {
+    value: number
+    referenceLineColor?: ColorDefinition
+    overThresholdLineColor?: ColorDefinition
+}
+
+export interface IntradayHeartRateChartProps {
+    previewState?: IntradayHeartRatePreviewState,
+    dataSource: DeviceDataV2Namespace,
+    aggregationOption: AggregationOption,
+    aggregationIntervalAmount: number,
+    aggregationIntervalType: "Hours" | "Minutes",
+    lineColor?: ColorDefinition,
+    thresholds?: IntradayChartThreshold[],
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+export default function IntradayHeartRateChart(props: IntradayHeartRateChartProps) {
+    const [data, setData] = useState<LineChartDataPoint[]>([]);
+    const [axis, setAxis] = useState<LineChartAxis>({ yRange: [0, 200], yIncrement: 30, xIncrement: props.aggregationIntervalAmount });
```

Comment:
I don't think yRange needs to be specified at all or yIncrement - in my branch it worked fine without having to specify these

Reply:
The value of having some default x or y axis in place is so the chart renders when there is no data. 
If nothing displays at all, the user may not be aware their device is is not connected. An empty chart could reinforce that the participant should connect their device

Reply:
I think we would want to mimic the "Daily data chart"'s no data state here which does not display a y-axis but does display the x-axis

Reply:
https://ui.mydatahelps.org/?path=/story/container-dailydatachart--no-data